api_definition/coepi_api_0.4.0.yml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.3
+
+info:
+  description: This is the API model for CoEpi Server
+  version: 0.4.0
+  title: CoEpi Server
+  license:
+    name: MIT License
+    url: https://github.com/Co-Epi/data-models/blob/master/LICENSE
+
+paths:
+  /tcnreport:
+    post:
+      summary: Submit symptom or infection report
+      description: Users submit symptom / infection reports and reveal the secret TCN Key in their application
+        as bytes in format 'rvk || tck_{j1-1} || le_u16(j1) || le_u16(j2) || memo'. Details of the TCN protocol
+        can be found here, https://github.com/TCNCoalition/TCN/blob/main/README.md
+
```

Comment:
Need to pull in the `x-amazon-apigateway-integration:` tags from the 0.3.0 version for all the endpoints here, so that this OpenAPI spec can be used with AWS tooling.

Reply:
will add